tests/mockServer/mock.js

```diff
@@ -227,6 +247,33 @@ function getRuns(req, res) {
     )
   }
 
+  if (req.query['label']) {
+    let key, value
+    ;[key, value] = req.query['label'].split('=')
```

Comment:
Doesn't _this_ work? 🤔 
```suggestion
    let [key, value] = req.query['label'].split('=')
```